rdr_service/dao/genomics_dao.py

```diff
@@ -74,6 +74,8 @@ class GenomicDaoMixin:
         GenomicJob.AW5_WGS_MANIFEST
     ]
 
+    exclude_states = [GenomicWorkflowState.IGNORE]
```

Comment:
So in the genomics dao there are over 20 occurrences of this, would you do a find and replace in that file to get all these conditions since we are introducing this pattern?